src/test/java/uk/gov/hmcts/reform/sscs/services/ccd/UpdateCcdHearingTypeTest.java

```diff
@@ -71,18 +78,18 @@ public void givenGapsHearingTypeUpdate_shouldUpdateExistingCcdCase(SscsCaseData
 
         SscsCaseData existingCcdCaseCorHearingType = SscsCaseData.builder()
             .appeal(Appeal.builder()
-                .hearingType("cor")
+                .hearingType(COR.getValue())
                 .build())
```

Comment:
Can you please add scenario for DOMICILIARY hearingtype in test

Reply:
It's there. Follow the variable gapsCaseDataDomiciliaryHearingType.
It's difficult to read the test Params, hence also why I added a comment in the tests for the new test I put in.